File: URI-py/1134.py
# -*- coding: utf-8 -*-
alcool = 0
gasolina = 0
diesel = 0
n = int(input())
while True:
    if n == 1:
        alcool += 1
    elif n == 2:
        gasolina += 1
    elif n == 3:
        diesel += 1
    elif n == 4:
        break
    n = int(input())
print("MUITO OBRIGADO\nAlcool: %d\nGasolina: %d\nDiesel: %d"%(alcool,gasolina,diesel))

File: URI-cpp/1134.cpp
#include <iostream>
#include <iomanip>
#include <vector>

using namespace std;

int main()
{
    int n,alcool=0,gasolina=0,diesel=0;

    while(true){
        cin >> n;
        if(n==1){
            alcool++;
        } else if(n==2) {
            gasolina++;
        } else if(n==3) {
            diesel++;
        } else if(n==4) {
            break;
        }
    }

    cout << "MUITO OBRIGADO" << endl << "Alcool: " << alcool << endl << "Gasolina: " << gasolina << endl << "Diesel: " << diesel << endl;

    return 0;
}